openmp/libomptarget/DeviceRTL/src/Debug.cpp

```diff
@@ -32,10 +32,10 @@ void __assert_fail(const char *expr, const char *file, unsigned line,
 void __assert_fail_internal(const char *expr, const char *msg, const char *file,
                             unsigned line, const char *function) {
   if (msg) {
-    PRINTF("%s:%u: %s: Assertion %s (`%s') failed.\n", file, line, function,
+    PRINTF("%s:%u: %s: Assertion %s (`%s`) failed.\n", file, line, function,
            msg, expr);
   } else {
-    PRINTF("%s:%u: %s: Assertion `%s' failed.\n", file, line, function, expr);
+    PRINTF("%s:%u: %s: Assertion `%s` failed.\n", file, line, function, expr);
```

Comment:
@jdoerfert doesn't really matter, but did you intend these to be apostrophes(') or grave accents (`)?

Reply:
I was probably going for apostrophes `'`. I might have been thinking latex: ``text'', who knows.